tests/TestFileUtilities.py

```diff
@@ -740,25 +732,18 @@ def test_wipe_name(self):
 
     def test_wipe_path(self):
         """Unit test for wipe_path()"""
+
         if None == os.getenv('ALLTESTS'):
-            print 'warning: skipping long test test_wipe_path() because environment variable ALLTESTS not set'
-            return
+            self.skipTest('warning: skipping long test test_wipe_path() because environment variable ALLTESTS not set')
+
         pathname = tempfile.gettempdir()
         for ret in wipe_path(pathname):
             # no idle handler
             pass
 
     def test_vacuum_sqlite3(self):
         """Unit test for method vacuum_sqlite3()"""
-
-        try:
-            import sqlite3
-        except ImportError, e:
-            if sys.version_info[0] == 2 and sys.version_info[1] < 5:
-                print "Warning: Skipping test_vacuum_sqlite3() on old Python"
-                return
-            else:
-                raise e
+        import sqlite3
```

Comment:
This is good. We don't need to make this PR any bigger with more of this, but FYI: I am cleaning out Python 2.5 specific code. I have some more of these clean ups planned after I merged your PR.